src/index.js

```diff
@@ -20,7 +20,12 @@ async function runTest(files, reporter) {
 
     try {
         let reporterStream = new TapMochaReporter(reporter);
-        await brs.execute([ ...rocaFiles, ...files], {
+        let allTestFiles = [...rocaFiles];
+        if (globalSetupFile) {
```

Comment:
I thought about also adding a check to see if it's a valid file path, but if it isn't, then `brs` will throw an error saying it can't find the file. So it would be redundant to have `roca` check as well.